File: Daily Coding/python/rem_dup.py
# Remove the duplicate element in an array
lst = []
length = int(input("Enter the length of the array: "))
print("Enter the array: ")
for i in range(length):
    temp = int(input())
    if temp not in lst:
        lst.append(temp)
for i in lst:
    print(i,end=" ")

File: Daily Coding/c/rem_dup.c
// Remove the duplicate element in an array
#include <stdio.h>
#include <stdlib.h>
int main()
{
    int *count = (int*)calloc(sizeof(int),256);
    int i, j, len, temp, a[50];
    printf("Enter the length of the array: ");
    scanf("%d",&len);
    printf("Enter the elements: ");
    for(i=0,j=0; i<len; i++)
    {
        scanf("%d",&temp);
        count[temp]++;
        if(count[temp] == 1)
            a[j++] = temp;
    }
    a[j] = '\0';
    for(i=0; a[i]; i++)
        printf("%d ",a[i]);
    return 0;
}
